test/unit/foreman/renderer/scope/report_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class ReportScopeTest < ActiveSupport::TestCase
+  setup do
+    source = Foreman::Renderer::Source::String.new(content: '')
+    @scope = Foreman::Renderer::Scope::Report.new(source: source)
+  end
+
+  describe '#report_render' do
+    test 'render simple values' do
+      @scope.report_row('Col1': 'Val1', 'Col2': 'Val2')
+      @scope.report_row('Col1': 'Val3', 'Col2': 'Val4')
+      expected_csv = "Col1,Col2\nVal1,Val2\nVal3,Val4\n"
+      assert_equal expected_csv, @scope.report_render(format: :csv)
+
+      expected_yaml = <<~OUT
+      ---
+      - Col1: Val1
+        Col2: Val2
+      - Col1: Val3
+        Col2: Val4
+      OUT
+      assert_equal expected_yaml, @scope.report_render(format: :yaml)
+    end
+
+    test 'render arrays' do
+      @scope.report_row('Col1': ['Val1','Val2'], 'Col2': 'Val3')
+      expected_csv = "Col1,Col2\n\"Val1,Val2\",Val3\n"
+      assert_equal expected_csv, @scope.report_render(format: :csv)
+
+      expected_yaml = <<~OUT
+      ---
+      - Col1:
+        - Val1
+        - Val2
+        Col2: Val3
+      OUT
+      assert_equal expected_yaml, @scope.report_render(format: :yaml)
+    end
+  end
```

Comment:
Could you please add a test that verifies empty report is rendered if `report_row` is not called but `report_renderˋ is?